README.md

```diff
@@ -108,10 +108,10 @@ Restate follows [Semantic Versioning](https://semver.org/).
 
 You can safely upgrade from a Restate `x.y` to `x.(y+1)` release without performing any manual data migration, as Restate performs an automatic data migration for you.
 
-To check the compatibility with the SDKs, check the respective READMEs:
+For SDK compatibility, refer to the supported version matrix in the respective READMEs:
```

Comment:
Perhaps generally, what I would write here is something along the lines of "get in touch with the respective SDK owners to check whether a new release should be made for the SDKs"

Reply:
Done!